src/System.Drawing.Common/src/System/Drawing/Gdiplus.cs

```diff
@@ -123,2380 +123,125 @@ private static void Initialize()
-            internal static extern int GdipSetPathGradientWrapMode(HandleRef brush,
-                                                                   int wrapmode);
-
-            [DllImport(ExternDll.Gdiplus, SetLastError = true, ExactSpelling = true, CharSet = CharSet.Unicode)]
-            internal static extern int GdipGetPathGradientWrapMode(HandleRef brush,
-                                                                   out int wrapmode);
-
-            [DllImport(ExternDll.Gdiplus, SetLastError = true, ExactSpelling = true, CharSet = CharSet.Unicode)]
-            internal static extern int GdipSetPathGradientTransform(HandleRef brush,
-                                                                    HandleRef matrix);
-
-            [DllImport(ExternDll.Gdiplus, SetLastError = true, ExactSpelling = true, CharSet = CharSet.Unicode)]
-            internal static extern int GdipGetPathGradientTransform(HandleRef brush,
-                                                                    HandleRef matrix);
-
-            [DllImport(ExternDll.Gdiplus, SetLastError = true, ExactSpelling = true, CharSet = CharSet.Unicode)]
-            internal static extern int GdipResetPathGradientTransform(HandleRef brush);
-
-            [DllImport(ExternDll.Gdiplus, SetLastError = true, ExactSpelling = true, CharSet = CharSet.Unicode)]
-            internal static extern int GdipMultiplyPathGradientTransform(HandleRef brush, HandleRef matrix, MatrixOrder order);
-
-            [DllImport(ExternDll.Gdiplus, SetLastError = true, ExactSpelling = true, CharSet = CharSet.Unicode)]
-            internal static extern int GdipTranslatePathGradientTransform(HandleRef brush, float dx, float dy, MatrixOrder order);
-
-            [DllImport(ExternDll.Gdiplus, SetLastError = true, ExactSpelling = true, CharSet = CharSet.Unicode)]
-            internal static extern int GdipScalePathGradientTransform(HandleRef brush, float sx, float sy, MatrixOrder order);
-
-            [DllImport(ExternDll.Gdiplus, SetLastError = true, ExactSpelling = true, CharSet = CharSet.Unicode)]
-            internal static extern int GdipRotatePathGradientTransform(HandleRef brush, float angle, MatrixOrder order);
-
-            [DllImport(ExternDll.Gdiplus, SetLastError = true, ExactSpelling = true, CharSet = CharSet.Unicode)]
-            internal static extern int GdipGetPathGradientFocusScales(HandleRef brush,
-                                                                      float[] xScale,
-                                                                      float[] yScale);
-
-            [DllImport(ExternDll.Gdiplus, SetLastError = true, ExactSpelling = true, CharSet = CharSet.Unicode)]
-            internal static extern int GdipSetPathGradientFocusScales(HandleRef brush,
-                                                                      float xScale,
-                   
```

Comment:
Is Initialize called from somewhere other than the static cctor?  If not, consider just merging it into the static cctor, so that code doesn't accidentally invoke it.